final_task/rss_reader/text_work_util.py

```diff
@@ -0,0 +1,147 @@
+import feedparser
+import html
+import logging
+import sys
+
+
+def fill_dict_with_info(dict_with_info: dict, all_info: object, limit: str) -> tuple:
+    if limit:
+
+        try:
+            if int(limit) > len(all_info.entries) or int(limit) < 0:
+                limit = int(limit)
+            else:
+                limit = all_info.entries[0:int(limit)]
+        except ValueError:
+            print('\n Should try only integer numbers')
+            sys.exit()
+
+    else:
+        limit = all_info.entries
+
+    try:
+        for enter in limit:
+            dict_with_info['Title'] += [enter.title]
+            dict_with_info['Date'] += [enter.published]
+            dict_with_info['Links'] += [enter.link]
+            dict_with_info['Info'] += [enter.summary]
```

Comment:
Do you really need to put objects inside lists here? Why not just simply use these objects?

`[enter.title]` and etc.